lib/rspec-puppet/adapters.rb

```diff
@@ -141,51 +80,72 @@ def setup_puppet(example_group)
                        end
                      end
 
-        if (rspec_manifest = RSpec.configuration.manifest)
-          manifest = rspec_manifest
-        else
-          manifest_paths = Puppet[:environmentpath].split(File::PATH_SEPARATOR).map do |path|
-            File.join(path, 'fixtures', 'manifests')
-          end
-
-          manifest = manifest_paths.find do |path|
-            File.exist?(path)
-          end
```

Comment:
It seems like this change completely removes use of `manifest` configuration setting. Am I missing where it's still used?

Reply:
Yes I think you're 100% correct.... Let me update this.